pkg/upgraders/upgradeable.go

```diff
@@ -0,0 +1,65 @@
+package upgraders
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/blang/semver"
+	"github.com/go-logr/logr"
+	configv1 "github.com/openshift/api/config/v1"
+	upgradev1alpha1 "github.com/openshift/managed-upgrade-operator/pkg/apis/upgrade/v1alpha1"
+)
+
+func (c *clusterUpgrader) IsUpgradeableVersion(ctx context.Context, logger logr.Logger) (bool, error) {
+	v := &configv1.ClusterVersion{}
+	currentVersion, err := c.cvClient.GetClusterVersion()
+	if err != nil {
+		return false, err
+	}
+	parsedCurrentVersion, err := semver.Parse(currentVersion.Status.History[0].Version)
```

Comment:
Can't always guarantee that the first element of CV's history is going to be the current version. For that reason, I recommend using the helper `GetCurrentVersion()` function instead and use it.